src/main/java/de/unistuttgart/overworldbackend/service/CourseService.java

```diff
@@ -232,6 +234,7 @@ private World cloneWorld(final World oldWorld, final String accessToken) {
             oldWorld.getStaticName(),
             oldWorld.getTopicName(),
             false,
+            oldWorld.getMinigameTasks().stream().anyMatch(minigame -> Minigame.isConfigured(minigame.getGame())),
```

Comment:
Why not directly copy the configured attribute?

Reply:
I got it wrong, it should get computed out of the new minigames, because if the backends aren't available, the coresponding worlds/dungeons aren't configured.

Reply:
Perhaps add a comment explaining exactly that to that line.